modules/storage/cosmos-db/modules/cassandra.bicep

```
@@ -0,0 +1,51 @@
+param cosmosDBAccountName string
+param enableServerless bool = false
+param keyspaceName string
+param cassandraTables array
+param autoscaleMaxThroughput int
+param manualProvisionedThroughput int
+
+resource cosmosDBAccount 'Microsoft.DocumentDB/databaseAccounts@2023-03-15' existing = {
```

```suggestion
resource cosmosDBAccount 'Microsoft.DocumentDB/databaseAccounts@2022-11-15' existing = {
```

Can you double check this API Version? I dont see it over here. https://learn.microsoft.com/en-us/azure/templates/microsoft.documentdb/allversions

My VSCode is picking '2022-11-15'

---

Actually `2023-03-15` version i shown on Azure portal but not yet available on bicep cmdline. Changed back all versions to `2022-11-15` for uniformity.